package.json

```
@@ -6,17 +6,25 @@
   "scripts": {
     "build": "NODE_ENV=production npx webpack --mode=production",
     "dev": "cross-env NODE_ENV=development npx webpack serve --mode=development",
-    "ts-check": "tsc --skipLibCheck --noEmit --project tsconfig.json"
+    "ts-check": "tsc --skipLibCheck --noEmit --project tsconfig.json",
+    "lint": "npx eslint . --ext .js,.jsx,.ts,.tsx",
+    "lint:fix": "eslint \"./src/**/*.{js,jsx,ts,tsx}\" --fix"
```

думаю тут можно привести скрипты к единому виду:

```suggestion
    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
    "lint:fix": "eslint ./src --ext .js,.jsx,.ts,.tsx --fix"
```